src/App.tsx

```diff
@@ -17,15 +17,17 @@ const Stack = createNativeStackNavigator();
 function App() {
   return (
     <ApolloProvider client={client}>
-      <NavigationContainer>
-        <Stack.Navigator>
-          <Stack.Screen
-            options={{ headerShown: false }}
-            name="Pokemon"
-            component={PokemonIndex}
-          />
-        </Stack.Navigator>
-      </NavigationContainer>
+      <NoConnection>
```

Comment:
:50_50:
Why you want to hide existing screen from user when internet is down? This will create bad UX if user has poor internet connection or will lost network for a moment